error-503.html

```diff
@@ -7,125 +7,159 @@
+        </ul>
+        <ul class="main-nav__account-menu account-menu">
+          <li><a href="https://www.nhs.uk/Personalisation/Login.aspx">Log in</a></li>
+          <li><a href="https://www.nhs.uk/Personalisation/Registration.aspx">Create an account</a></li>
+        </ul>
       </div>
-    </header>
+    </nav>
+    <!-- endnoindex -->
+  </header>
+
+
+      <main id="content" role="main" class="util-nofocus">
```

Comment:
I don't think the pharmacy finder specific messaging and navigation should be included in the site wide error page.

Reply:
Removed